Yosemite/YosemiteTests/Stores/ReceiptStoreTests.swift

```diff
@@ -47,7 +47,9 @@ final class ReceiptStoreTests: XCTestCase {
     }
 
     func test_print_calls_print_in_service() throws {
-        let mockParameters = try XCTUnwrap(MockPaymentIntent.mock().receiptParameters())
+        // Given
+        let mockIntent = PaymentIntent.fake().copy(charges: [.fake().copy(paymentMethod: .cardPresent(details: .fake()))])
```

Comment:
As previous comment for the `Charge.fake` here. TestData doesn't need to be shared between the two test classes (though it could be if you prefer... but I think it would be better to keep those enums scoped to each test class.)

Reply:
Similar to above, updated in https://github.com/woocommerce/woocommerce-ios/pull/6817/commits/ad5326d54f3573d792c31bfb0e5bf02ebe712bc7